wallet/wallet.go

```diff
@@ -3413,8 +3436,9 @@ func (w *Wallet) SendOutputs(outputs []*wire.TxOut, keyScope *waddrmgr.KeyScope,
 	// been confirmed.
 	createdTx, err := w.CreateSimpleTx(
 		keyScope, account, outputs, minconf, satPerKb,
-		coinSelectionStrategy, false,
-	)
+		coinSelectionStrategy, false, WithCustomSelectUtxos(
+			selectedUtxos))
```

Comment:
nit: closing parenthesis should be on next line(s):
```
	createdTx, err := w.CreateSimpleTx(
		keyScope, account, outputs, minconf, satPerKb,
		coinSelectionStrategy, false, WithCustomSelectUtxos(
			selectedUtxos,
		),
	)
```

Or, if you prefer this way:
```
	createdTx, err := w.CreateSimpleTx(
		keyScope, account, outputs, minconf, satPerKb,
		coinSelectionStrategy, false,
		WithCustomSelectUtxos(selectedUtxos),
	)
```